lib/python/vdm/server/venv_starter.py

```diff
@@ -209,8 +209,15 @@ def start_virtual_environment(arr, packages=None, verbose=False):
                         [venv_base])
             else:
                 build_venv = venv_version != version
+
+        packages = [os.path.join(G.base_dir, 'lib/python/vdm/requirements.txt')]
         if build_venv:
             _build_virtual_environment(venv_dir, version, packages)
+        else:
+            if packages:
+                for package in packages:
```

Comment:
Do you need this for loop here? packages variable will now contain the path to requirements.txt and  you can call it directly without the for loop, correct?